spyder/plugins/projects/widgets/tests/test_projectdialog.py

```diff
@@ -8,12 +8,20 @@
 Tests for projectdialog.py
 """
 
-# Test library imports
+# Standard library imports
 import pytest
+try:
+    from unittest.mock import Mock
+except ImportError:
+    from mock import Mock  # Python 2
+
+# Third party imports
+import os
```

Comment:
``os`` is in the standard library, so it should go immediately above ``pytest`` and you can remove the ``# Third party imports`` line and extra linebreak entirely.